docs/core/introduction.md

```diff
@@ -1,105 +1,68 @@
 ---
-title: .NET (and .NET Core) - introduction and overview
+title: .NET - introduction and overview
 description: Learn about .NET (and .NET Core). .NET is a free, open-source development platform for building many kinds of apps.
 ms.date: 11/15/2023
 ms.custom: "updateeachrelease"
 ---
 # What is .NET? Introduction and overview
```

Comment:
We have more polished and approachable "What is .NET" page at https://dotnet.microsoft.com/en-us/learn/dotnet/what-is-dotnet

Should this page be called something else - to avoid competition between the two pages in search results and to make it clear that this one is more in-depth explanation?

Reply:
That makes sense to me. Having the same page names between docs and the marketing site seems counter-productive.

Maybe just call this page ".NET overview"?

Reply:
@jkotas Are we able to see what keywords/terms lead to entry on this page? I want to see if it's worth considering something like "What is .NET and what can I build with it?"

Reply:
"What is .NET" search points to the marketing site. This page is much further down.

Reply:
I updated the title to "Introduction to .NET". That's pretty clearly what the doc is, at least per the filename.